wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -482,4 +534,12 @@ private[wallet] trait TransactionProcessing extends WalletLogger {
         } yield utxos
     }
   }
+
+  private[wallet] def getTransactionsToBroadcast: Future[
+    Vector[Transaction]] = {
+    for {
+      unconfirmedUtxos <- spendingInfoDAO.findAllPendingConfirmation
```

Comment:
So this doesn't really make sense to me, why would we want to re-broadcast transactions that could possibly have 1 confirmation? 

Do we need to add a new `TxoState` that represents a broadcast but not confirmed _at all_ state? 

It seems our TxoState's currently can't represent what is required here?

Reply:
Ah yes, seems so